lib/src/protocol/kawa_h1/mod.rs

```diff
@@ -505,8 +503,10 @@ impl<Front: SocketHandler, L: ListenerHandler + L7ListenerHandler> Http<Front, L
                 _ => (),
             }
 
+            let response_length_known = self.response_stream.body_size != kawa::BodySize::Empty;
```

Comment:
the usual practice is Sōzu is this one:

```
use some_crate::Enum;

// ...

let _ = Enum::Variant;
```

I suggest you replace `kawa::BodySize::Empty`  with `BodySize::Empty`, and import `BodySize` in the kawa use statement.

Reply:
For now we qualify everything that comes from `kawa`, nothing is imported. This was originally done to make the frontier between Sozu and Kawa as explicit and easy to "follow" as possible.